apps/prime.c

```diff
@@ -13,10 +13,13 @@
 #include "progs.h"
 #include <openssl/bn.h>
 
+#define BUFSIZE 2048
```

Comment:
FWIW, I'd double this and add 1 to make it `4098`, so that in hex the max input value is then 16K bits (followed by a terminating LF and NUL).